jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/applications/WindowsVariantEndpoint.java

```diff
@@ -138,7 +138,7 @@ public Response updateWindowsVariation(
             windowsVariant.setName(updatedWindowsVariant.getName());
             windowsVariant.setDescription(updatedWindowsVariant.getDescription());
             variantService.updateVariant(windowsVariant);
-            return Response.noContent().build();
+            return Response.ok(windowsVariant).build();
```

Comment:
I wonder if must not retrieve the correct subtype like we do in the `GET` , https://github.com/matzew/aerogear-unifiedpush-server/blob/ReturnVariantDetailsOnUpdate/jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/applications/WindowsVariantEndpoint.java#L96